specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2022-10-01/machineLearningServices.json

```diff
@@ -4723,6 +4723,27 @@
         "adminPublicKey": {
           "type": "string",
           "description": "Specifies the SSH rsa public key file as a string. Use \"ssh-keygen -t rsa -b 2048\" to generate your SSH key pairs."
+        },
+        "adminPublicKeys": {
+          "type": "object",
+          "x-nullable": true,
+          "description": "List of keys (with friendly names) authorized for SSH. If specified, the value of 'adminPublicKey' is ignored.",
+          "additionalProperties": {
+            "$ref": "#/definitions/SshPublicKeyReference"
+          }
+        }
+      }
+    },
+    "SshPublicKeyReference": {
+      "type": "object",
+      "properties": {
+        "source": {
+          "type": "string",
+          "default": "local"
+        },
+        "data": {
```

Comment:
>data [](http://example.com/codeflow?start=9&length=4)

do you need to annotate with: `"x-ms-secret": true` if this property is used in a PUT / PATCH consider marking as a secret, properties marked as secrets are not meant to be returned in a PUT or GET (single or list) calls